src/jenkins/common/Openenclave.groovy

```diff
@@ -142,27 +142,52 @@ def WinCompilePackageTest(String dirName, String buildType, String hasQuoteProvi
     cleanWs()
     checkout scm
     dir(dirName) {
-        bat """
-            vcvars64.bat x64 && \
-            cmake.exe ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${buildType} -DBUILD_ENCLAVES=ON -DHAS_QUOTE_PROVIDER=${hasQuoteProvider} -DLVI_MITIGATION=${lviMitigation} -DLVI_MITIGATION_SKIP_TESTS=${lviMitigationSkipTests} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCPACK_GENERATOR=NuGet -Wdev ${extra_cmake_args.join(' ')} && \
-            ninja.exe && \
-            ctest.exe -V -C ${buildType} --timeout ${timeoutSeconds} && \
-            cpack.exe -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY && \
-            cpack.exe && \
-            (if exist C:\\oe rmdir /s/q C:\\oe) && \
-            nuget.exe install open-enclave -Source %cd% -OutputDirectory C:\\oe -ExcludeVersion && \
-            set CMAKE_PREFIX_PATH=C:\\oe\\open-enclave\\openenclave\\lib\\openenclave\\cmake && \
-            cd C:\\oe\\open-enclave\\openenclave\\share\\openenclave\\samples && \
-            setlocal enabledelayedexpansion && \
+        /*
+        In simulation mode, some samples should not be ran or should run simulation mode. 
+        For items that should be skipped, see items appended to SAMPLES_LIST under the IF statement with OE_SIMULATION in:
+        https://github.com/openenclave/openenclave/blob/master/samples/test-samples.cmake#L54
+        For items that should run in simulation mode, check sample Makefiles for target `simulate`
+        SIMULATION_SKIP is a "list" of samples to skip in simulation mode.
+        SIMULATION_TEST is a "list" of samples to run in simulation mode.
+        */
+        bat(
+            returnStdout: false,
+            returnStatus: false,
+            script: """
+            call vcvars64.bat x64
+            cmake.exe ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${buildType} -DBUILD_ENCLAVES=ON -DHAS_QUOTE_PROVIDER=${hasQuoteProvider} -DLVI_MITIGATION=${lviMitigation} -DLVI_MITIGATION_SKIP_TESTS=${lviMitigationSkipTests} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCPACK_GENERATOR=NuGet -Wdev ${extra_cmake_args.join(' ')} || exit !ERRORLEVEL!
+            ninja.exe || exit !ERRORLEVEL!
+            ctest.exe -V -C ${buildType} --timeout ${timeoutSeconds} || exit !ERRORLEVEL!
+            cpack.exe -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY || exit !ERRORLEVEL!
+            cpack.exe || exit !ERRORLEVEL!
+            if exist C:\\oe rmdir /s/q C:\\oe
+            nuget.exe install open-enclave -Source %cd% -OutputDirectory C:\\oe -ExcludeVersion
+            set CMAKE_PREFIX_PATH=C:\\oe\\open-enclave\\openenclave\\lib\\openenclave\\cmake
+            setlocal EnableDelayedExpansion
+            set SIMULATION_SKIP="\\attested_tls\\attestation\\"
```

Comment:
file-encryptor is skipped but also has a `simulate` target in the Makefile. Right now this PR has it running ninja simulate and tests have passed: https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Windows/detail/Azure-Windows/3035/pipeline/14/

Do we want to skip or use `ninja simulate`?

Reply:
If it works, we should run ninja simulate.